Documentation/Signoffs/SensorSysem.md

```diff
@@ -0,0 +1,106 @@
+# Sensor Subsystem Signoff
+## FUNCTION OF THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The sensor subsystem is the actuator of our Air Quality design. These sensors will analyze and receive data from the outside and 
```

Comment:
Delete all the fluff: first sentence, third sentence, and most or all of the fourth sentence.